packages/react-core/src/components/DataList/DataListActionBreakpoints.ts

```diff
@@ -0,0 +1,18 @@
+export enum DataListActionModifiers {
+  hidden = 'hidden',
+  visible = 'visible'
+}
+
+export enum DataListActionBreakpoints {
+  md = 'Md',
+  lg = 'Lg',
+  xl = 'Xl',
+  '2xl' = '2Xl'
+}
+
+export interface DataListActionBreakpointMod {
+  /** The attribute to modify  */
+  modifier: keyof typeof DataListActionModifiers;
```

Comment:
Does this type show in the docs anyway? I'm pretty that gives the right type but I couldn't find when I went to verify, sorry if I'm overlooking it..

Reply:
Umm probably I forgot to add it to the props...
Thanks for noticing! :smiley_cat: